agent/functional_tests/tests/functionaltests_windows_test.go

```diff
@@ -266,14 +268,20 @@ func TestTelemetry(t *testing.T) {
 	time.Sleep(waitMetricsInCloudwatchDuration)
 
 	cwclient := cloudwatch.New(session.New(), aws.NewConfig().WithRegion(*ECS.Config.Region))
-	err = VerifyMetrics(cwclient, params, true)
+	err, _ = VerifyMetrics(cwclient, params, true)
 	assert.NoError(t, err, "Before task running, verify metrics for CPU utilization failed")
 
 	params.MetricName = aws.String("MemoryUtilization")
-	err = VerifyMetrics(cwclient, params, true)
+	err, _ = VerifyMetrics(cwclient, params, true)
 	assert.NoError(t, err, "Before task running, verify metrics for memory utilization failed")
 
-	testTask, err := agent.StartTask(t, "telemetry-windows")
+	cpuNum := runtime.NumCPU()
+
+	tdOverrides := make(map[string]string)
+	// Set the container cpu percentage 25%
+	tdOverrides["$$$$CPUSHARE$$$$"] = strconv.Itoa(cpuNum * 1024 / 4)
```

Comment:
Can you `* 0.25` instead of `/ 4` so that it's clearer with respect to your comment?

Can you add a constant for 1024?  `cpuSharesPerCore` would be reasonable.